string    speech
float32[] words_histogram
string[]  words_dictionary
string[]  complete_words
string    frames
